CMakeLists.txt

```diff
@@ -232,9 +231,7 @@ else()
 endif()
 
 
-if(DETECT_PKG_CONFIG)
-  find_package(PkgConfig)
-endif()
+find_package(PkgConfig)
```

Comment:
What's the need of finding `pkgconfig`?  Since .pc files are being installed regardless in your change, it seems to me that this line is unnecessary.